js/src/main/scala/org/widok/bindings/Bootstrap/package.scala

```diff
@@ -84,6 +84,10 @@ package object Bootstrap {
     case object Danger extends Style { val cssSuffix = "danger" }
   }
 
+  case class Code(contents: View*) extends Widget[Code] { 
+    val rendered = DOM.createElement("pre", contents) 
```

Comment:
In HTML, there is already a tag called `code`, so it would be a good idea to use it here. You can create a second `case class` for `pre` and call it `Preformatted`. Please also add an alias to the [package file](https://github.com/widok/widok/blob/master/js/src/main/scala/org/widok.scala#L74).